### DISCUSSION TITLE
Material Symbols not working with Vite plugin

### DISCUSSION THREAD

**Discussion by TristanRib** [quasarframework/quasar#16752] [General - Components / Directives / etc] [answered]
Hi !

I'have been trying to use material symbols outlined for a long time but i can't make it.
I'm using quasar with vue cli :

```js
import quasarIconSet from 'quasar/icon-set/material-symbols-outlined'
import '@quasar/extras/material-symbols-outlined/material-symbols-outlined.css'

app.use(Quasar, {
    iconSet: quasarIconSet,
    config: {
        dark: 'auto' // or Boolean true/false
    }
});
```

I'm looking for the icon named "swords"

```html
  <div style="background-color: red">
    <q-icon name="swords"/>
  </div>
```

![image](https://github.com/quasarframework/quasar/assets/116078722/5cb9e132-fc52-4bfc-b65d-1c6003dfb33d)
Nothing appears.

But as soon as i handly replace the class"material-icons" with "material-symbols-outlined" on the icon component, it works
![image](https://github.com/quasarframework/quasar/assets/116078722/dfe2d12c-9c51-4171-9362-f6e0d7c31f12)
![image](https://github.com/quasarframework/quasar/assets/116078722/f6c0bd23-ddb4-4194-8f6d-1621961684b3)

I have tried the same configuration with fontawesome, it's working just fine with any icons from the set.
```js
import iconSet from 'quasar/icon-set/fontawesome-v6'
import '@quasar/extras/fontawesome-v6/fontawesome-v6.css'

app.use(Quasar, {
    iconSet: iconSet,
    config: {
        dark: 'auto' // or Boolean true/false
    }
});
```

```html
  <div style="background-color: red">
    <q-icon name="fa-solid fa-khanda"/>
  </div>
```
![image](https://github.com/quasarframework/quasar/assets/116078722/72184316-f84e-47ac-a563-b2129d65c144)

It may be a rookie mistake but i can't find it and it's frustrating.

I basically just want quasar to use material symbols outlined ;(

Thanks for helping !

**Comment by vmornar** [comment]
I have the same issue, could someone give a hint? Using all latest vite & quasar libraries. With just 
`import '@quasar/extras/material-icons/material-icons.css', all is working fine, except, of course, missing icons that I need

I have:
import iconSet from 'quasar/icon-set/material-symbols-outlined.js'
import '@quasar/extras/material-symbols-outlined/material-symbols-outlined.css'
...
```
app.use(Quasar, {
  plugins: {
    LocalStorage,
    SessionStorage,
    Dialog,
    Notify
  }, // import Quasar plugins and add here
  lang: {}, //langQ.default,
  iconSet: iconSet,
  config: {
    dark: false
  },
  extras: [
    'material-symbols-outlined',
  ]
})
```

**Comment by rstoenescu** [comment, marked as answer]
Hi,

This is incorrect usage. You need to prefix the name of the icon according to the set that you are loading. Example: sym_o_thumb_up.

Prefixes are clearly described in docs: QIcon page, "Web Fonts" section. https://quasar.dev/vue-components/icon#webfont-usage
